src/tree/localProject/ConfigTreeItem.ts

```diff
@@ -3,22 +3,31 @@
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
+import { ThemeIcon } from "vscode";
 import { AzExtTreeItem, TreeItemIconPath } from "vscode-azureextensionui";
-import { treeUtils } from "../../utils/treeUtils";
-import { ConfigGroupTreeItem } from "./ConfigGroupTreeItem";
+import { BuildConfig, GitHubConfigGroupTreeItem } from "./ConfigGroupTreeItem";
 
-export class ConfigTreeItem extends AzExtTreeItem {
-    public static contextValue: string = 'azureStaticConfig';
-    public contextValue: string = ConfigTreeItem.contextValue;
-    public readonly label: string;
-    public parent: ConfigGroupTreeItem;
+export class GitHubConfigTreeItem extends AzExtTreeItem {
+    public static contextValue: string = 'azureStaticGitHubConfig';
+    public contextValue: string = GitHubConfigTreeItem.contextValue;
+    public commandId: string = 'staticWebApps.openYAMLConfigFile';
+    public commandArgs: unknown[];
+    public readonly buildConfig: string;
+    public buildConfigValue: string;
+    public parent: GitHubConfigGroupTreeItem;
 
-    public constructor(parent: ConfigGroupTreeItem, label: string) {
+    public constructor(parent: GitHubConfigGroupTreeItem, buildConfig: BuildConfig, buildConfigValue: string) {
         super(parent);
-        this.label = label;
+        this.buildConfig = buildConfig;
+        this.buildConfigValue = buildConfigValue;
+        this.commandArgs = [this.parent, this.buildConfig];
+    }
+
+    public get label(): string {
+        return `${this.buildConfig}="${this.buildConfigValue}"`;
```

Comment:
We should probably implement a `refreshImpl` on this tree item.  I can see CTI saying the values don't update on an edit.